packages/node/src/utils/sync-helper.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright 2020-2022 OnFinality Limited authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import { blake2AsHex } from '@polkadot/util-crypto';
```

Comment:
This adds a polkadot dependency

Reply:
This is fine, its just crypto utils and we're using some polkadot deps for MMR/POI